src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/network/sdk/AzureVirtualNetworkOperationSDK.java

```diff
@@ -104,15 +113,15 @@ private void subscribeVirtualNetworkCreation(Observable<Indexable> virtualNetwor
                 .subscribe();
     }
 
-    public AzureGetVirtualNetworkRef doGetInstance(String resourceName, AzureUser azureUser)
+    public AzureGetVirtualNetworkRef doGetInstance(String resourceName, String orderName, AzureUser azureUser)
             throws FogbowException {
 
         Network network = getNetwork(resourceName, azureUser);
 
         VirtualNetworkInner virtualNetworkInner = network.inner();
         String provisioningState = virtualNetworkInner.provisioningState();
         String id = virtualNetworkInner.id();
-        String name = network.name();
+        String name = orderName;
```

Comment:
Obtain the resource name from the resource tag, as follows: network.tags().get(AzureConstants.TAG_NAME);

Reply:
Done.